core/Plugin/Report.php

```diff
@@ -918,21 +918,42 @@ public function getThirdLeveltableDimension()
             return null;
         }
```

Comment:
That check is actually unneeded, as it's performed in `getNthLeveltableDimension` anyway.